packages/perseus/src/types.ts

```diff
@@ -341,6 +341,18 @@ export type PerseusDependencies = {
     isMobile: boolean;
 };
 
+/**
+ * The modern iteration of Perseus Depedndencies. These dependencies are
+ * provided to Perseus through its entrypoints (for example:
+ * ServerItemRenderer) and then attached to the DependenciesContext so they are
+ * available anywhere down the React render tree.
+ *
+ * Prefer using this type over `PerseusDependencies` when possible.
+ */
+export type PerseusDependenciesV2 = {
```

Comment:
Just for clarification, are we planning on using only this iteration of Perseus dependencies in the future and we're planning on slowly phasing out the old PerseusDependencies or are there cases where we would need to use the old version?

Reply:
> are we planning on using only this iteration of Perseus dependencies in the future
Yes. 
> we're planning on slowly phasing out the old PerseusDependencies
Yes!  

For now, it's a bit clunky and there are a few places that we couldn't access the React Context for dependencies. Until those areas are fixed, we need to keep the old PerseusDependencies global. All other uses can migrate.